Sources/AuthFoundation/JWT/ClaimContainer.swift

```diff
@@ -16,13 +16,13 @@ import Foundation
 ///
 /// > Note: This does not apply to JWT, which while it contains claims, it has a different format which includes headers and signatures.
 public protocol ClaimContainer: JSONDecodable {
-    var rawValue: [String:Any] { get }
+    var payload: [String:Any] { get }
 }
 
 extension ClaimContainer {
```

Comment:
```suggestion
extension ClaimsContainer {
```
Seems that this is a container with the whole claims payload. If I'm misunderstanding, then disregard 😄

Reply:
It's a container of one or more claims.  There's an enum called `Claim` that represents a single claim.  So perhaps this should be named `ClaimCollection` instead.